homeassistant/components/vesync/__init__.py

```diff
@@ -48,6 +50,11 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     hass.data[DOMAIN] = {}
     hass.data[DOMAIN][VS_MANAGER] = manager
 
+    coordinator = VeSyncDataCoordinator(hass, manager)
+
+    # Store coordinator at domain level since only single integration instance is permitted.
```

Comment:
I think it would be nice if we could open up the integration to allow multiple instances

Reply:
I think implementing that would be outside the scope of this PR. I think multiple integrations would only be possible if multiple logins are supported.

Reply:
Migrating location of coordinator later would add more work so it is a fair point.

Reply:
Oh definitely different PR, but I still wanted to mention it